frontend/app/page.tsx

```diff
@@ -109,6 +150,9 @@ export default function SignUp() {
                   id="password"
                   autoComplete="new-password"
                   type={showPassword ? "text" : "password"}
+                  onChange={(event) =>
+                    setPassword(event.target.value)
+                  }
```

Comment:
**nitpick**: Why is this formatted differently from the previous anonymous functions?

Reply:
autoformatted but ill just make it 1 line